### PR TITLE
fix: move mjpeg acceleration option into trickplay panel

### LINKED ISSUE #5363: "Allow encoding in MJPEG format (used during trickplay generation)" not being saved
**Describe The Bug**
The button "Allow encoding in MJPEG format (used during trickplay generation)" gets reset after each reload of the page. It is not clear to me if it is actually being enabled, and as such is a web issue, or if it's not being saved properly, in which case it is probably a server issue.

**Steps To Reproduce**
1. Go to Settings
2. Click on Playback
3. Scroll down to Encoding Format Options
4. Check "Allow encoding in MJPEG format (used during trickplay generation)"
5. Save.
6. Reload.
7. Button should have been toggled back.

**Expected Behavior**
"Allow encoding in MJPEG format (used during trickplay generation)" should remain enabled after saving.

**Logs**
Not aware of any relevant log entries.

**System (please complete the following information):**
Jellyfin Web.

**Additional Context**
N/A

### PR DESCRIPTION
**Pull request by gnattu**
**Changes**

This is a followup of #5326. This option is very trickplay specific and should not belongs to general encoding options. This PR moves the option into the trickplay panel and changed the option texts to be more specific and less confusing.

**Issues**
This change is related to jellyfin/jellyfin#11229

Fixes #5363

### CODE REVIEW

**Review by nyanmisaka — APPROVED** [review]

File: src/strings/en-us.json [hunk 1]
```diff
@@ -1648,10 +1648,10 @@
     "ForeignPartsOnly": "Forced/Foreign parts only",
     "HearingImpairedShort": "HI/SDH",
     "LabelIsHearingImpaired": "For hearing impaired (SDH)",
-    "AllowMjpegEncoding": "Allow encoding in MJPEG format (used during trickplay generation)",
     "Trickplay": "Trickplay",
-    "LabelTrickplayAccel": "Enable hardware acceleration",
-    "LabelTrickplayAccelHelp": "Make sure to enable 'Allow MJPEG Encoding' in Transcoding if your hardware supports it.",
+    "LabelTrickplayAccel": "Enable hardware decoding",
+    "LabelTrickplayAccelEncoding": "Enable hardware accelerated MJPEG encoding",
+    "LabelTrickplayAccelEncodingHelp": "Currently only available on QSV and VAAPI, this option has no effect on other hardware acceleration methods.",
```
[the remark this answers is not in the document]
Comment (gnattu) [reply src/strings/en-us.json:1654]: Then modify the text in the future. 

VideoToolbox also provides MJPEG encoder but it is not implemented in ffmpeg.

MJPEG in this small size should not be a too big problem for modern CPUs anyway.

**Review by thornbill — CHANGES_REQUESTED** [review]
The `LinkTrickplayAcceleration` file and any related translations should be removed also.

**Comment by gnattu** [comment]
> The `LinkTrickplayAcceleration` file and any related translations should be removed also.

Done

**Review by thornbill — COMMENTED** [review]
Only the en-us translation file should be modified. Weblate handles the rest.

**Comment by gnattu** [comment]
> Only the en-us translation file should be modified. Weblate handles the rest.

Done. The one in en-US was already removed before your comments so I thought it was needed to cleanup the leftovers in other languages as well.

**Review by nielsvanvelzen — APPROVED** [review]

**Review by thornbill — APPROVED** [review]

**Pull request merged**